assets/src/edit-story/components/library/mediaLibrary.js

```diff
@@ -76,33 +194,74 @@ function MediaLibrary( { onInsert } ) {
 						<Spinner />
 					}
 				</Title>
-				<UploadButton mediaType={ mediaType } onClose={ onClose } onSelect={ onSelect } />
+				<UploadButton
+					onClose={ onClose }
+					onSelect={ onSelect }
+				/>
 			</Header>
 
+			<div style={ {
```

Comment:
Should we use styled-components here for consistency?

Reply:
Fixed in https://github.com/ampproject/amp-wp/pull/3907/commits/d02b0b72525a12bbc64d5ee2412ec3ea958a0665